modules/navigation.js

```diff
@@ -0,0 +1,15 @@
+export default function displayList() {
+  document.querySelectorAll('.section').forEach((section) => {
+    if (!(section.id === 'book-list')) section.style.display = 'none';
+  });
+  const navBar = document.querySelector('.navBar');
+  navBar.addEventListener('click', (e) => {
+    if (e.target.classList.contains('navLink')) {
+      const anchorHref = e.target.href.split('#')[1];
+      document.querySelectorAll('.section').forEach((section) => {
+        if (section.id === anchorHref) section.style.display = 'flex';
+        else section.style.display = 'none';
+      });
+    }
+  });
+}
```

Comment:
- [ ] Kindly make sure to change this function to the `Arrow` function in order to meet the milestone requirements for using ES6 javascript arrow functions.

Reply:
change function to the[` Arrow`](https://github.com/Kel-nana/Awesome-Books-Modules/blob/6bd1920a0af1939f64cf79ec2adb44f22f206af4/modules/navigation.js#L1-L16) function